dartsim/src/SDFFeatures.cc

```diff
@@ -534,7 +534,7 @@ Identity SDFFeatures::ConstructSdfLink(
     const Identity &_modelID,
     const ::sdf::Link &_sdfLink)
 {
```

Comment:
I don't see any tests written for `SDFFeatures::ConstructSdfLink`. Should we add one?

Reply:
Unless there's a specific reason to do so, IMO, this is covered by the `ConstructWorld` tests in `SDFFeatures_TEST.cc`.